M2/Macaulay2/packages/GKMManifolds.m2

```diff
@@ -0,0 +1,1468 @@
+-*-------------------------------------------------------------------------------------------
+   Copyright 2020 Christopher Eur and Ritvik Ramkumar.
+
+   You may redistribute this file under the terms of the GNU General Public
+   License as published by the Free Software Foundation, either version 2 of
+   the License, or any later version.
+-------------------------------------------------------------------------------------------*-
+
+
+
+newPackage("GKMManifolds",
+	Version => "0.1",
+	Date => "August 1, 2020",
+	Authors => {
+	    {Name => "Chris Eur",
+       	     Email => "chriseur@stanford.edu",
+       	     HomePage => "https://web.stanford.edu/~chriseur"},
+	    {Name => "Ritvik Ramkumar",
+	     Email => "ritvik@math.berkeley.edu",
+	     HomePage => "https://math.berkeley.edu/~ritvik"}
+	    },
+	Headline => "a package for computations with GKM manifolds and moment graphs",
+	HomePage => "https://github.com/chrisweur/GKMManifolds",
+	PackageExports => {"Graphs", "Matroids", "NormalToricVarieties"},
+	AuxiliaryFiles => true,
+	DebuggingMode => true
```

Comment:
please turn off debugging mode (that's only for developers of the package)

Reply:
done